src/Pages/Product/Productdetails.scss

```
@@ -162,7 +182,27 @@
   //<-- Center Explanation -->
   .useExplanation {
     display: flex;
-    height: 100vh;
+    height: 940px;
+
+    .productVideoContainer {
+      width: calc(100% * 1 / 2);
+      top: 0;
+      left: 0;
+      animation: 1000ms 1ms slide-right--100 cubic-bezier(0.32, 0.24, 0.15, 1)
+        forwards;
+      transform: translate(-100%, 0);
+      background-color: $lightColor;
+      height: 100%;
```

```suggestion
      width: calc(100% * 1 / 2);
      height: 100%;
      top: 0;
      left: 0;
      animation: 1000ms 1ms slide-right--100 cubic-bezier(0.32, 0.24, 0.15, 1)
        forwards;
      transform: translate(-100%, 0);
      background-color: $lightColor;
```
1. width와 height는 붙어 있는게 좋아보입니다
2. top과 left가 relative나 absolute 없이 쓰이고 있습니다

---

피드백 감사합니다! 컴포넌트에서 삭제한 JXS인데 scss 삭제를 못했네요! 삭제하겠습니다:)